### DISCUSSION TITLE
Request Tablist

### DISCUSSION THREAD

**Discussion by BlindMag** [Zrips/CMI#470] [Feature Requests]
**Description of issue or feature request:**
Bungeecord support for tablist -

If possible we would like to see separate sections per server with usernames under those server names in the tablist.

**Comment by TheWranger** [comment]
That sounds like adding an entire new system to how CMI handles tablists.

**Comment by BlindMag** [comment]
I just noticed it wasn't the author who responded. Eh, late night! I'd love to see this support bungee, especially that it kinda gets stuck between servers and wants to hold onto the other server's tablist when you move over servers.

**Comment by Zrips** [comment]
Possible, but not a priority at this point.
